app-editors/emacs/emacs-25.2.recipe

```diff
@@ -0,0 +1,127 @@
+SUMMARY="The famous text editor"
+DESCRIPTION="An extensible, customizable, free/libre text editor — and more.
+At its core is an interpreter for Emacs Lisp, a dialect of the Lisp \
+programming language with extensions to support text editing.
+
+* Content-aware editing modes, including syntax coloring, for many file types.
+* Complete built-in documentation, including a tutorial for new users.
+* Full Unicode support for nearly all human scripts.
+* Highly customizable, using Emacs Lisp code or a graphical interface.
+* An entire ecosystem of functionality beyond text editing, including a \
+project planner, mail and news reader, debugger interface, calendar, and more.
+* A packaging system for downloading and installing extensions."
+HOMEPAGE="https://gnu.org/s/emacs/"
+COPYRIGHT="2001-2017 Free Software Foundation, Inc."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.artfiles.org/gnu.org/emacs/emacs-$portVersion.tar.xz"
+CHECKSUM_SHA256="59b55194c9979987c5e9f1a1a4ab5406714e80ffcfd415cc6b9222413bc073fa"
+PATCHES="emacs-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 ?x86"
+
+PROVIDES="
+	emacs$secondaryArchSuffix = $portVersion
+	cmd:ctags = $portVersion
```

Comment:
Please rename the "ctags" binary to "ctags.emacs", as it conflicts with the binary from the ctags package.